frontend/package.json

```diff
@@ -27,6 +30,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "prettier": "prettier --write .",
+    "relay": "relay-compiler --src ./src --schema ./schema.graphql --extensions ts tsx",
```

Comment:
Note to self: tässä annettu `--extensions`-vipu taitaa olla turha, koska `relay.config.js`:ssä on tehty samat määrittelyt.

Reply:
Muutkin vivut olivat vähän turhia, koska samat asiat oli määritelty configissa... Otin pois.